### PR TITLE
EFRS-760: model headers renaming

### PR DESCRIPTION
**Pull request by omorgunova-pl**

### CODE REVIEW

**Comment by omorgunova-pl** [comment]
I have read the CLA Document and I hereby sign the CLA

**Review by daffy- — DISMISSED** [review]

File: ui/src/app/features/create-dialog/create-dialog.component.html [hunk 1]
```diff
@@ -14,7 +14,7 @@
   ~ permissions and limitations under the License.
   -->
 
-<h1 mat-dialog-title>{{'common.create' | translate}} {{data.entityType}}</h1>
```
Comment (isobolev) [inline ui/src/app/features/create-dialog/create-dialog.component.html:17]: why {{data.entityType}} was removed?

File: ui/src/app/features/delete-dialog/delete-dialog.component.html [hunk 2]
```diff
@@ -14,8 +14,7 @@
   ~ permissions and limitations under the License.
   -->
 
-<h1 mat-dialog-title>{{'org_users.delete_dialog.title' | translate}}
-  {{data.entityType === 'system-user' ? 'user' : data.entityType }}</h1>
```
Comment (isobolev) [inline ui/src/app/features/delete-dialog/delete-dialog.component.html:18]: same as above. Delete dialog is used not only for model

File: ui/src/app/features/edit-dialog/edit-dialog.component.html [hunk 3]
```diff
@@ -14,7 +14,7 @@
   ~ permissions and limitations under the License.
   -->
 
-<h1 mat-dialog-title>Edit {{data.entityType}}</h1>
```
Comment (isobolev) [inline ui/src/app/features/edit-dialog/edit-dialog.component.html:17]: entityType

File: ui/src/app/features/create-dialog/create-dialog.component.html (hunk 1 above)
Comment (omorgunova-pl) [reply ui/src/app/features/create-dialog/create-dialog.component.html:17]: > why {{data.entityType}} was removed?

entityType: 'model', this interpolation gives the word "model", and as it was mentioned in the task, we have to remove it from the dialog.

**Pull request closed without merging**